legistar/events.py

```diff
@@ -202,12 +202,15 @@ def _scrapeWebCalendar(self):
             # Make the dict key (name, datetime.datetime), and add it.
             response = self.get(event['iCalendar']['url'], verify=False)
             web_scraper._check_errors(response)
-            event_time = web_scraper.ical(response.text).subcomponents[0]['DTSTART'].dt
-            event_time = pytz.timezone(self.TIMEZONE).localize(event_time)
-
-            key = (event['Name']['label'],
-                   event_time)
-            web_info[key] = event
+            # "response.text" sometimes returns an empty string
```

Comment:
The seems like something that should be handled by _check_errors. If not, why not?

Reply:
I considered that, but _check_errors is a nice, concise single-responsibility function. I did not want to convolute it...
With that said, I think we could add a conditional for an empty string and return a different status code, perhaps, a 520? Something like:

```
if not response.text:
        response.status_code = 520
        raise scrapelib.HTTPError(response)
```